opencog/query/InitiateSearchCB.cc

```diff
@@ -653,7 +653,7 @@ bool InitiateSearchCB::variable_search(PatternMatchEngine *pme)
 	bool all_clauses_are_evaluatable = true;
 	for (const Handle& cl : clauses)
 	{
-		if (0 < _pattern->evaluatable_terms.count(cl)) continue;
+		if (0 < _pattern->evaluatable_holders.count(cl)) continue;
 		all_clauses_are_evaluatable = false;
```

Comment:
EinsteinUTest recently slowed down from 10 seconds to 95 seconds, on my machine. It is either due to pull req #666 or possibly to the above change on other changes in this pull req.  See bug #676